fabric-provider-api-v1/src/main/java/net/fabricmc/fabric/api/provider/v1/BlockApiProviderAccess.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.provider.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.entity.BlockEntity;
+import net.minecraft.block.entity.BlockEntityType;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import net.fabricmc.fabric.impl.provider.BlockApiProviderAccessImpl;
+
+/**
+ * See {link ApiProviderAccess}. This subclass is for {@code Block} game objects.
```

Comment:
There is an `@see` tag you can place at the bottom of the javadoc for the class.

Reply:
Aware of that.  I wanted this to be more prominent (at the top) because I'm not copying the docs from the super type and they are important. Unfortunately `{@inheritDoc}` only seems to work for class members.